sig-community/charter.md

```diff
@@ -0,0 +1,41 @@
+# SIG Community
+
+## Scope
+SIG Community is responsible for fostering, growing, and sustaining Cilium’s open source community. We focus on contributor experience, community engagement, and governance processes to ensure a welcoming and productive project. Our efforts include onboarding new contributors, maintaining contribution infrastructure and governance, and supporting events and advocacy efforts that promote Cilium’s adoption.
+
+We collaborate with other SIGs to streamline contribution workflows, grow the contributor community, and strengthen Cilium’s ecosystem through outreach and engagement.
+
+## In scope
+
+### Community Growth & Contributor Experience
+- Defining and improving the contributor onboarding process  
+- Maintaining contributor governance  
+- Encouraging diversity and inclusion in the contributor community
+- Improving transparency around project decision making processes
+
```

Comment:
Hearing feedback from contributors and taking actions if most agrees

Reply:
sounds good, I'll add that